apiserver/admin_test.go

```diff
@@ -967,7 +968,7 @@ func (s *loginSuite) TestLoginUpdatesLastLoginAndConnection(c *gc.C) {
 	model := s.ControllerModel(c)
 	modelUser, err := model.State().UserAccess(names.NewUserTag("bobbrown"), model.ModelTag())
 	c.Assert(err, jc.ErrorIsNil)
-	when, err := model.LastModelConnection(modelUser.UserTag)
+	when, err := userService.LastModelLogin(context.Background(), modelUser.UserTag.Name(), coremodel.UUID(model.UUID()))
```

Comment:
I checked with Joe about this, we decided that the user should go first because this is on the user service. I see your point though, I did have it the other way round originally